src/System.ComponentModel.Composition/src/Microsoft/Internal/GenerationServices.cs

```diff
@@ -225,9 +223,8 @@ public static void AddLocalToLocalDictionary(this ILGenerator ilGenerator, Local
 
         /// Generates the code to get the type of an object and store it in a local
         /// <param name="ilGenerator"></param>
-        /// <param name="dictionary"></param>
-        /// <param name="key"></param>
-        /// <param name="value"></param>
+        /// <param name="exception"></param>
```

Comment:
I don't think I would classify this as "fixing" the suppression. It is rather working around the suppression without actually providing the documentation.

Reply:
Definitely it's not a "fix", however there are a lot of files with such "stubs" for comments. When we remove global warning suppression from Directory.Build.props we can fine-tune these warnings on project / file level.